src/main/java/harvard/Storage.java

```diff
@@ -0,0 +1,49 @@
+
+package harvard;
+import java.util.ArrayList;
+/**
+ * Represents a list of tasks.
+ */
+
+public class Storage {
+    /**
+     * The file path.
+     */
+    private String filePath;
+    /**
+     * Constructs a Storage object.
+     * @param filePath The file path.
```

Comment:
I think there is usually a space between the params and the header of the comment?